src/commands/component.js

```diff
@@ -86,18 +96,9 @@ class ComponentCommand extends AbstractCommand {
    */
   _deleteComponent(name) {
     const configPath = this.getConfigPath(name);
-    if (configPath) {
-      const configFiles = this.listAllEnvConfig(configPath);
-
-      return Promise.all(configFiles.map(it => fse.remove(it))).then(() => {
-        this.logger.info(`Done for terrahub component: '${name}'`);
-
-        return Promise.resolve();
-      });
-    }
+    const configFiles = this.listAllEnvConfig(configPath);
 
-    this.logger.warn(`Terrahub component with provided name: '${name}' doesn't exist`);
-    return Promise.resolve();
+    return Promise.all(configFiles.map(it => fse.remove(it))).then(() => Promise.resolve(name));
```

Comment:
It is redundant to return component name
```suggestion
    return Promise.all(configFiles.map(it => fse.remove(it)));
```